Unix/pal/strings.c

```diff
@@ -348,10 +306,9 @@ int wcscasecmp(const wchar_t* s1, const wchar_t* s2)
 
     return (int)((unsigned int)*s1 - (unsigned int)*s2);    
 }
-#endif
 
```

Comment:
Can you check to see if anyone actually uses wcsncasecmp()?  If not, just delete it.  Without wchar, there shouldn't be anyone calling this function.